esmvaltool/cmorizers/data/cmor_config/ESACCI-LST.yml

```diff
@@ -27,6 +27,6 @@ variables:
     file_day: 'ESACCI-LST-L3C-LST-MODISA-0.05deg_1MONTHLY_DAY-'
     file_night: 'ESACCI-LST-L3C-LST-MODISA-0.05deg_1MONTHLY_NIGHT-'
     # plan to make it possible to change MODISA here but only placeholders in the python there for now
-    start_year: 2003
+    # start_year: 2003
     # 2003 Only has data from July onwards for Aqua MODIS
-    end_year: 2018
+    # end_year: 2018
```

Comment:
why is this commented out (and the one above too)? If it's not needed then pls remove, if it's optional maybe add an inline comment say why it's optional :+1:

Reply:
I'll merge after this is looked at :beer:

Reply:
Thanks @valeriupredoi! Should be cleaner now